### ISSUE TITLE
💬 Discussion | Add Mission Statment

### ISSUE THREAD

**Issue by jkhgvfgvsth** [privacytoolsIO/privacytools.io#1021]
**Description**: Add Mission Statement / Goal
**Why?**: What is privacytools.io's long term goal?
**Ideas**: Feel free to comment them below.
I wanted to propose:
`Ensuring a list of tools made to long-term respect users privacy, freedom, and security.'

**Comment by nitrohorse** [comment]
Hmm, doesn’t the header sub-text encapsulate the mission already?

> privacytools.io provides services, tools and knowledge to protect your privacy against global mass surveillance.

**Comment by five-c-d** [comment]
Yes, that is effectively the mission-statement.  But I think that the thrust of what jkhgv is suggesting, is changing the emphasis and the wording.  

* M1: "list of tools made to long-term respect users privacy, freedom, and security." 
* M2:  "Tools to respect your privacy, your freedom, and your security -- over the long-term" 

Would be my rewrite of their suggestion into more idiomatic form.  That would drop explicit mention of 'services' (as well as of 'knowledge'), add mention of 'freedom' and 'security', and drop explicit mention of global mass surveillance.  

See the related discussion here == #966 which discussed whether privacyToolsIO could be a project that recommends tools, while getting money from the tool-creators... and also, discussed whether privacyToolsIO should be involved with providing services/tools, as opposed to concentrate on ranking/listing what other tool-creators provide.  

Not every listing is a libre-licensed tool, so it would probably not be correct to add "freedom" to the mission-statement.  Similarly, I don't think that the mission-statement should really be **about** political questions like whether Trisquel is superior to Debian on overall **moral** grounds, for the same reason that I don't think the listings should take any position on temperature of the troposphere.  #1036 and #798 specifically, about which @Mikaela and myself disagree :-)   I think the site should concentrate only upon directly improving privacy and thwarting surveillance, full stop.  Indirectly changing the world in other ways, is out of scope, in other words.  

Point being, I think I disagree with jkhgv's insertion of "freedom" into the mission.  I think it is *important* that endusers have freedom, but I don't think they can get that from the privacyToolsIO listings, which are specifically about *privacy*.  And are pieces of software (and if that new category ever is merged maybe hardware), whereas for freedom you actually need societal and/or governmental changes, waaay beyond what mere bits & bytes can conceivably accomplish.  Personally, I would argue for this as the subtitle-slash-mission-statement:  

* M0:  "privacyTools.IO provides services, tools and knowledge to protect your privacy against global mass surveillance."
* M3:  "privacyTools.IO lists tools which protect your privacy against mass surveillance." 

My version 'M3' of the mission-statement, removes mention of internally-provided services -- it is still fine to LIST those because they are "tools which protect your privacy" obviously.  But, with this change in emphasis, like other tools by 3rd parties in the listings, internal services would also henceforth have to go through the process of getting discussed/vetted/etc (rather than being "automatically" added by virtue of being tools/services that the listing-creators are personally involved with).  

I also remove the word "global" and the word "knowledge" because I think those are implied by the other words.  Mass surveillance is the badguy here, and global mass surveillance is a type of mass surveillance.  The tools-listings are not useful, unless they come with the knowledge of how to use those tools... or at least, hyperlinks to that knowledge :-) 

***I*** would also be in favor of adding something at the end, like 

* M4:  "privacyTools.IO lists tools which protect your privacy against mass surveillance, thereby improving freedom and security for all of us in the long run."  

But I'm not sure that IS the mission :-)  So although I think it would be a cool thing to add, I'm leery of adding things without LOTS of discussion amongst the core team, to make sure the additions are 100% supported and 100% clear.  Is freedom one of the long-term goals?  If so, how should it be mentioned in the mission-statement?  Or should it just be implicitly there, in the form of the listings themselves and what they recommend (freedom-promoting libre-licensed tools for the vast majority of the categories)... and NOT explicitly mentioned in the mission-statement?

**Comment by blacklight447-ptio** [comment]
Changed issue name as we already have a mission statement
